.github/workflows/linux_build_test.yml

```diff
@@ -66,20 +70,46 @@ jobs:
         OUTPUT_DETAILS: detailed
         VALIDATE_CSHARP: true
         IGNORE_GITIGNORED_FILES: true
-        FILTER_REGEX_INCLUDE: "${{ inputs.workdir }}"
+        FILTER_REGEX_INCLUDE: "${{ inputs.workdir }}/.*"

```

Comment:
Same here. ${{ inputs.workdir }}/.* might not work for this. I think you need to remove the dot from the filter in order for the filter to work correctly.